frontend/package.json

```diff
@@ -1,79 +1,75 @@
+		"root": "file:..",
+		"rxjs": "~7.5.0",
+		"tslib": "^2.3.0",
 		"wait-on": "^6.0.1",
-		"web-vitals": "^2.1.4"
+		"zone.js": "~0.11.4"
 	},
-	"devDependencies":
-	{
+	"devDependencies": {
+		"@angular-builders/jest": "13.0.3",
+		"@angular-devkit/build-angular": "~13.3.2",
+		"@angular-eslint/builder": "13.2.0",
+		"@angular-eslint/eslint-plugin": "13.2.0",
+		"@angular-eslint/eslint-plugin-template": "13.2.0",
+		"@angular-eslint/schematics": "13.2.0",
+		"@angular-eslint/template-parser": "13.2.0",
+		"@angular/cli": "^13.3.3",
+		"@angular/compiler-cli": "~13.3.0",
+		"@briebug/jest-schematic": "^4.0.0",
+		"@cypress/schematic": "^1.6.0",
 		"@isnotdefined/eslint-config": "4.4.0",
-		"@testing-library/jest-dom": "^5.16.2",
-		"@testing-library/react": "^12.1.4",
-		"@testing-library/user-event": "^13.5.0",
+		"@isnotdefined/stylelint-config": "^3.2.0",
 		"@types/cypress": "1.1.3",
-		"@types/express": "^4.17.13",
 		"@types/jest": "^26.0.24",
-		"@types/node": "^17.0.21",
-		"@types/react": "^17.0.40",
-		"@types/react-dom": "^17.0.13",
-		"@wojtekmaj/enzyme-adapter-react-17": "^0.6.6",
-		"cross-env": "7.0.3",
-		"cypress": "^9.5.2",
-		"eslint": "8.11.0",
-		"jest": "^26.6.3",
-		"kill-port-process": "3.0.1",
-		"stylelint": "14.6.0",
-		"@isnotdefined/stylelint-config": "^3.2.0",
-		"ts-jest": "^26.5.6",
-		"typescript": "^4.6.2"
-	},
-	"scripts":
-	{
-		"start": "react-scripts start",
-		"start:local": "cross-env-shell REACT_APP_ENV=local npm run start",
-		"start:dev": "cross-env-shell REACT_APP_ENV=dev npm run start",
-		"start:stage": "cross-env-shell REACT_APP_ENV=stage npm run start",
-		"start:prod": "cross-env-shell REACT_APP_ENV=prod npm run start",
-		"build": "react-scripts build",
-		"build:local": "cross-env-shell REACT_APP_ENV=local npm run build",
-		"build:dev": "cross-env-shell REACT_APP_ENV=dev npm run build",
-		"build:stage": "cross-env-shell REACT_APP_ENV=stage npm run build",
-		"build:prod": "cross-env-shell REACT_APP_ENV=prod npm run build",
-		"test": "react-scripts test",
-		"test:local": "cross-env-shell REACT_APP_ENV=local npm run test",
-		"test:dev": "cross-env-shell REACT_APP_ENV=dev npm run test",
-		"test:stage": "cross-env-shell REACT_APP_ENV=stage npm run test",
-		"test:prod": "cross-env-shell REACT_APP_ENV=prod npm run test",
-		"e2e": "cypress run",
-		"e2e:local": "cross-env-shell BROWSER=none CYPRESS_APP_ENV=local npm run e2e",
-		"e2e:dev": "cross-env-shell BROWSER=none CYPRESS_APP_ENV=dev npm run e2e",
-		"e2e:stage": "cross-env-shell BROWSER=none CYPRESS_APP_ENV=stage npm run e2e",
-		"e2e:prod": "cross-env-shell BROWSER=none CYPRESS_APP_ENV=prod npm run e2e",
-		"eject": "react-scripts eject",
-		"lint": "eslint \"{cypress,src}/**/*.{ts,tsx}\" --cache && stylelint \"src/**/*.css\" --cache",
-		"fix": "eslint \"{cypress,src}/**/*.{ts,tsx}\" --fix && stylelint \"src/**/*.css\" --fix ",
-		"wait-for-serve": "wait-on tcp:3000 tcp:8000"
+		"@typescript-eslint/eslint-plugin": "5.17.0",
```

Comment:
I think both typescript-eslint dependencies are not needed as  they are part of the isnotdefined/eslint-config ...